src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotation.java

```diff
@@ -155,6 +163,11 @@ public static GencodeFuncotationBuilder getBuilder() {
 
     //==================================================================================================================
 
+    @Override
+    public Allele getAltAllele() {
+        return Allele.create(tumorSeqAllele2.getBytes(), false);
```

Comment:
tumorSeqAllele2 is not an ideal name, that's a maf thing though isn't it?

Reply:
Yup.  This is an official field in MAF...